mineinabyss-components/src/main/kotlin/com/mineinabyss/components/guilds/GuildConfig.kt

```diff
@@ -0,0 +1,14 @@
+package com.mineinabyss.components.guilds
+
+import kotlinx.serialization.Serializable
+
+
+/**
+ * @property maxLength The maximum length of a guild name.
+ * @property bannedWords List for blocking certain words from guild names.
+ */
+@Serializable
+class GuildConfig (
+    val maxLength: Int = 20,
+    val bannedWords: List<String> = emptyList(),
```

Comment:
Just general note, `list()` is smart enough to default to empty list, there's no need to be more explicit.

Reply:
It actually isnt and complains if it is empty